R/utilities.R

```diff
@@ -282,3 +278,70 @@ expand_patients <- function(patients, all_pts) {
     )
     return(patients)
 }
+
+
+
+#' Decompose Patients into Relevant Components
+#'
+#' This function takes in a character vector or list of patients and decomposes it into a
+#' structured format.
+#'
+#' The primary use of this function is to correctly setup indexing variables for
+#' predicting survival quantities (see [`predict(SurvivalSamples)`][SurvivalSamples-class])
+#'
+#' @param patients (`character` or `list`) patient identifiers. If `NULL` will be set to `all_pts`.
```

Comment:
```suggestion
#' @param patients (`character` or `list`)\cr patient identifiers. If `NULL` will be set to `all_pts`.
```
also elsewhere (at least this is the convention I used so far when you were away)